### ISSUE TITLE
GM_notification.onclick is not working on macOS version of Firefox

### ISSUE THREAD

**Issue by rinsuki** [Tampermonkey/tampermonkey#710]
## Expected Behavior

1. Install example script
1. open any https sites
1. pop desktop notification "test please click"
1. click notification
1. website says `onclick is available!! :tada:`

(works on chrome)

## Actual Behavior

website did'nt says `onclick is available!! :tada:`.

## Specifications

- ~~Chromium~~ Firefox: 68.0b1 (Developer Edition)
- TM: 4.9.5941
- OS: macOS 10.14.4

## Script

```js
// ==UserScript==
// @name         GM_notification.onclick test
// @namespace    https://rinsuki.net/
// @version      0.1
// @description  try to take over the world!
// @author       rinsuki
// @match        https://*/*
// @grant        GM_notification
// ==/UserScript==

(function() {
    'use strict';

    // Your code here...
    GM_notification("please click", "test please click", null, () => {
        alert("onclick is available!! :tada:")
    })
})();
```

**Comment by beardhatcode** [comment]
I tested with the same script, and it did work as expected (a notification was shown and clicking it showed an alert). The only difference I have with you is the OS and the version of Firefox. Can you try and reproduce it on Mac with The Firefox Nightly version? If the problem persist, you may want to rename this issue to mention that it is a Mac only problem.

My specs:
TM: v4.9.5941
Browser: Firefox Nightly 69.0a1 (2019-05-29) (64-bit)
Window manger: SwayWM
OS: Linux 5.1.5 - arch1-2-ARCH

**Comment by rinsuki** [comment]
@beardhatcode 

I tested with Firefox Nightly 69.0a1(2019-06-02), but alert is not shown.

> Just to be sure, can you check if the following snippet shows a notification and handles the click event properly (paste this in the developer console):

```
(()=>{var n = new Notification("Test", {}); n.onclick = (e)=>{alert("ok")}})();
```

this script works well.

![image](https://user-images.githubusercontent.com/6533808/58796166-81c4ea80-8637-11e9-9561-945fcbc07a52.png)

**Comment by derjanb** [comment]
Hi, sorry for the delay. This issue should be fixed at [TM Beta 4.10.6105](https://www.tampermonkey.net/?browser=firefox).

**Comment by rinsuki** [comment]
I confirmed fix! thank you <3
